src/test/kotlin/org/jetbrains/research/transformations/PythonTransformationSetupTest.kt

```diff
@@ -0,0 +1,29 @@
+package org.jetbrains.research.transformations
+
+import PythonTransformationsTest
+import com.github.gumtreediff.gen.python.PythonTreeGenerator
+import com.github.gumtreediff.tree.TreeContext
+import org.junit.jupiter.api.Test
+
+/*
+* Just check if the parsers work correct
+* */
+internal class PythonTransformationSetupTest: PythonTransformationsTest() {
+
+    private fun getTreeContext(): TreeContext {
+        val srcFile = javaClass.getResource("source.py").path
+        return PythonTreeGenerator().generateFromFile(srcFile)
+    }
+
+    @Test
+    fun checkParserSetup() {
+        getTreeContext()
```

Comment:
Shouldn't we assert something here? like 'no errors were raised'?

Reply:
The test will fail if an error occurs

Reply:
I did not find a better way for this case

Reply:
You can try to use `assertDoesNotThrow` which is available since JUnit 5. It's a more accurate way to show what exactly do you test.

Reply:
Wow! Thank you a lot! I did not know about this function!